### DISCUSSION TITLE
Regarding my account problem

### DISCUSSION THREAD

**Discussion by Manu577228** [community/community#114363] [New to GitHub]
Hello,

I am encountering issues with my account. I am unable to push, pull, or clone any repositories, and my profile is not visible. I am currently preparing for interviews, and most of my links are in production. Whenever I am trying to do something it says "you are marked as a spam and cannot access third party devices"

I would greatly appreciate it if you could resolve this issue as soon as possible.

Thank you for your attention to this matter.

Best regards,
Manu Bharadwaj BN

**Comment by benogoulart** [comment]
Hi,

If your account has been marked as spam, the restriction itself can only be reviewed by GitHub Support. Unfortunately, the community can't remove spam flags or access account details.

Based on what you've described, the issues you're experiencing—being unable to push, pull, or clone repositories, your profile not being visible, and receiving the message that you're marked as spam—are all consistent with an account restriction that only GitHub Support can investigate and resolve.

Since you've already contacted Support, I recommend replying to your existing support ticket rather than creating a new one. In your reply, mention that:

* You're unable to push, pull, or clone any repositories.
* Your GitHub profile is not publicly visible.
* You're receiving the message that your account has been marked as spam and cannot access third-party devices.
* This is impacting your interview preparation because your portfolio and repository links are currently in use.

GitHub Support is the only team that can review the spam flag and restore access if the restriction was applied in error.

In the meantime, could you clarify one detail?

* Are you able to sign in to your GitHub account normally, or are you experiencing login issues as well?

That information may help determine whether this is solely a spam restriction or if there's an additional account issue involved.
